src/main/java/uk/gov/hmcts/reform/waworkflowapi/external/taskservice/CamundaClient.java

```diff
@@ -11,8 +12,12 @@
     url = "${camunda.url}"
 )
 public interface CamundaClient {
-    @PostMapping(value = "/decision-definition/key/getTask/evaluate", produces = MediaType.APPLICATION_JSON_VALUE)
-    List<GetTaskDmnResult> getTask(DmnRequest<GetTaskDmnRequest> requestParameters);
+    @PostMapping(value = "/decision-definition/key/getTask_{jurisdiction}_{caseType}/evaluate", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
since this is a lookup key have we checked if this is case sensitive since it maps directly to the decision table?

if not should we ensure the naming convention is correct? 
for `jurisdiction` and `caseType`, we could have enums hopefully making it easier to maintain, if we were to add new jurisdictions and case types.

Reply:
At the moment we were hoping to bring teams on board without making code changes so didn't want an enum. We did talk about having another DMN table where you registered a jurisdiction and case type but decided not to for now.